Examples/ObjectiveC/ShapeCollectionFeatureExample.m

```diff
@@ -0,0 +1,66 @@
+#import "ShapeCollectionFeatureExample.h"
+@import Mapbox;
+
+NSString *const MBXExampleShapeCollectionFeature = @"ShapeCollectionFeatureExample";
+
+@interface ShapeCollectionFeatureExample () <MGLMapViewDelegate>
+@property (nonatomic) MGLMapView *mapView;
+@end
+
+@implementation ShapeCollectionFeatureExample
+
+- (void)viewDidLoad {
+    [super viewDidLoad];
+    
+    self.mapView = [[MGLMapView alloc] initWithFrame:self.view.bounds styleURL:[MGLStyle lightStyleURLWithVersion:9]];
+    self.mapView.autoresizingMask = UIViewAutoresizingFlexibleWidth | UIViewAutoresizingFlexibleHeight;
+    
+    [self.mapView setCenterCoordinate:CLLocationCoordinate2DMake(38.897435, -77.039679) zoomLevel:12 animated:NO];
+    self.mapView.delegate = self;
+    
+    [self.view addSubview:self.mapView];
+}
+
+- (void)mapView:(MGLMapView *)mapView didFinishLoadingStyle:(MGLStyle *)style {
+    
+    // Parse the GeoJSON data.
+    dispatch_async(dispatch_get_global_queue(DISPATCH_QUEUE_PRIORITY_DEFAULT, 0), ^{
+        NSURL *url = [NSURL URLWithString: @"https://api.mapbox.com/datasets/v1/mapbox/cj004g2ay04vj2xls3oqdu2ou/features?access_token=pk.eyJ1IjoibWFwYm94IiwiYSI6ImNpemc0YWlpNzAwcXUyd21ldDV6OWpxMGwifQ.A92RQZpwUgtGtCmdSE4-ow"];
```

Comment:
Following up on https://github.com/mapbox/ios-sdk-examples/pull/69#discussion_r104875443, we should replace the Datasets API usage with local GeoJSON file usage. That way we can link to the GeoJSON file, allowing the developer to inspect its contents and compare its structure to the code here.

Otherwise, if we intend to use this opportunity to advertise the Datasets API, let’s provide a comment linking out to more information about it. This would be the first time many developers will’ve encountered that API. Opening a URL to a data set on the Mapbox account wouldn’t tell the developer anything about how they can create and edit their own data sets.